src/Cake.Core/ICakeEnvironment.cs

```diff
@@ -43,9 +43,15 @@ public interface ICakeEnvironment
         /// <summary>
         /// Gets all environment variables.
         /// </summary>
-        /// <returns>The environment variables as IDictionary&lt;string, string&gt;. </returns>
+        /// <returns>The environment variables as IDictionary&lt;string, string&gt;.</returns>
         IDictionary<string, string> GetEnvironmentVariables();
 
+        /// <summary>
+        /// Gets all command line arguments.
+        /// </summary>
+        /// <returns>The command line arguments as IDictionary&lt;string, string&gt;.</returns>
+        IDictionary<string, string> GetArguments();
```

Comment:
- [ ] Move API contract to [ICakeArguments](https://github.com/cake-build/cake/blob/v1.1.0/src/Cake.Core/ICakeArguments.cs#L12)
- [ ] Return type should be `IDictionary<string, ICollection<string>>` to allow for multiple values for the same argument

Reply:
Done